hack/deploy/kubedb-catalog.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: kubedb.com/v1alpha1
+kind: PostgresVersion
+metadata:
+  name: "9.6"
+  labels:
+    app: kubedb
+spec:
+  db:
+    image: "${KUBEDB_DOCKER_REGISTRY}/postgres:9.6"
+  exporter:
+    image: "${KUBEDB_DOCKER_REGISTRY}/pg-operator:0.8.0"
+  tools:
+    image: "${KUBEDB_DOCKER_REGISTRY}/postgres-tools:9.6"
+
+---
+apiVersion: kubedb.com/v1alpha1
+kind: PostgresVersion
+metadata:
+  name: "9.6.7"
+  labels:
+    app: kubedb
+spec:
+  db:
+    image: "${KUBEDB_DOCKER_REGISTRY}/postgres:9.6.7"
+  exporter:
+    image: "${KUBEDB_DOCKER_REGISTRY}/pg-operator:0.8.0"
```

Comment:
`operator:0.8.0`

The exporter is part of the combined operator . 

- https://github.com/kubedb/postgres/blob/master/pkg/docker/images.go#L8
- https://github.com/kubedb/operator/blob/master/pkg/exporter/exporter_handler.go#L80